priv/riak_kv.schema

```diff
@@ -185,25 +185,7 @@
 
 {translation,
  "riak_kv.aae_throttle_limits",
- fun(Conf) ->
-   %% Grab all of the possible names of tiers so we can ensure that
-   %% both mailbox_size and delay are included for each tier.
-   TierNamesM = cuttlefish_variable:fuzzy_matches(["anti_entropy", "throttle", "$tier", "mailbox_size"], Conf),
-   TierNamesD = cuttlefish_variable:fuzzy_matches(["anti_entropy", "throttle", "$tier", "delay"], Conf),
-   TierNames = lists:usort(TierNamesM ++ TierNamesD),
-   Throttles = lists:sort(lists:foldl(
-     fun({"$tier", Tier}, Settings) ->
-         Mbox = cuttlefish:conf_get(["anti_entropy", "throttle", Tier, "mailbox_size"], Conf),
-         Delay = cuttlefish:conf_get(["anti_entropy", "throttle", Tier, "delay"], Conf),
-         [{Mbox - 1, Delay}|Settings]
-     end, [], TierNames)),
-   case Throttles of
-       %% -1 is a magic "minimum" bound and must be included, so if it
-       %% isn't present we call it invalid
-       [{-1,_}|_] -> Throttles;
-       _ -> cuttlefish:invalid("anti_entropy.throttle tiers must include a tier with mailbox_size 0")
-   end
-end
+ riak_core_throttle:create_limits_translator_fun("anti_entropy", "mailbox_size")
```

Comment:
Make sure to switch out `riak_kv.aae_throttle_kill_switch` to `riak_kv.aae_throttle_enabled` here and in the riak_core_throttle module so that we can stop the cognitive dissonance.